### PR TITLE
Swap Org–User relationship

### PR DESCRIPTION
**Pull request by tudoramariei**

### CODE REVIEW

File: backend/accounts/migrations/0010_user_organization.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def reverse_user_organization_relationship(apps, _):
+    UserModel = apps.get_model("accounts", "User")
+
+    for user in UserModel.objects.all():
+        if user_org := user.organization:
+            user.organization = user_org
```
Comment (danniel) [inline backend/accounts/migrations/0010_user_organization.py:12]: shouldn't we save this?

File: backend/hub/models.py [hunk 2]
```diff
@@ -498,10 +496,13 @@ def save(self, *args, **kwargs):
 
     def create_owner(self):
```
Comment (danniel) [inline backend/hub/models.py:497]: Do we still need to automatically create an owner for an Organization?

**Review by danniel — APPROVED** [review]

File: backend/hub/forms.py [hunk 3]
```diff
@@ -290,7 +290,7 @@ def __init__(self, *args, **kwargs):
         self.initial["org"] = self.organization.id
 
     def clean_org(self):
-        return self.user.orgs.first().id
+        return self.user.organization.id
```
[the remark this answers is not in the document]
Comment (tudoramariei) [reply backend/hub/forms.py:293]: TBD for a later date. This is an issue across the whole app and we should tackle it all at once.

File: backend/hub/models.py (hunk 2 above)
Comment (tudoramariei) [reply backend/hub/models.py:497]: Leaving this as it is for now. I feel it's just a little bit risky.

**Pull request merged**